src/Util/AdminObjectAclData.php

```diff
@@ -303,7 +303,7 @@ public function isOwner()
     /**
      * Gets security handler.
      *
-     * @return SecurityHandlerInterface
+     * @return AclSecurityHandlerInterface
```

Comment:
Maybe we should add a check in the constructor or in this method that `$this->admin->getSecurityHandler` returns an `AclSecurityHandlerInterface`.

And then you use in all other methods of this class `$this-> getSecurityHandler` instead of `$this->admin->getSecurityHandler`.

Reply:
yeah, thanls